bin/docker.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+set -e
```

Comment:
I would recommend using `set -o errexit -o nounset -o pipefail` in all bashfiles, not just `-e`.